src/vs/workbench/contrib/terminalContrib/suggest/browser/terminalSuggestAddon.ts

```diff
@@ -369,6 +374,32 @@ export class SuggestAddon extends Disposable implements ITerminalAddon, ISuggest
 		};
 	}
 
+	private _getFontInfo(): ISimpleSuggestWidgetFontInfo {
+		const font = this._terminalConfigurationService.getFont(dom.getActiveWindow());
+		let lineHeight: number = font.lineHeight;
+		const fontSize: number = font.fontSize;
+		const fontFamily: string = font.fontFamily;
+		const letterSpacing: number = font.letterSpacing;
+		const fontWeight: string = this._configurationService.getValue('editor.fontWeight');
+
+		if (lineHeight <= 1) {
+			// Scale so icon shows by default
+			lineHeight = fontSize < 16 ? Math.ceil(fontSize * 1.5) : fontSize;
+		} else if (lineHeight <= 8) {
+			lineHeight = fontSize * lineHeight;
+		}
```

Comment:
Is this how the editor was scaling line height? We want large fontSizes to look the same in editor and terminal

Reply:
yes it is

Reply:
no, line height looks a little different. i'll have to see what they do